tool/tests/test.py

```diff
@@ -193,21 +193,6 @@ def test_forall_implies_2(self):
         ge_y = map.forall(state, lambda k, v: v >= Y)
         self.assertSolver(state, ~(ge_y & (Y >= X)) | ge_x)
 
-    # TODO why can't we prove this?
```

Comment:
it's useful to have examples of things we can't prove; perhaps they can be skipped instead of entirely deleted? I imagine Python's unittest has a way to do that

Reply:
now actively checking that the related tests do throw